packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts

```diff
@@ -17,7 +17,7 @@ export const getStyles = (props: IGroupFooterStyleProps): IGroupFooterStyles =>
         position: 'relative',
         padding: '5px 38px',
         selectors: {
-          [`:global(.${classNames.link}`]: {
+          [`& :global(.${classNames.link})`]: {
```

Comment:
Can we add a screener test for this please :)

Reply:
I'll just remove it then :)